slash/runner.py

```diff
@@ -120,10 +119,6 @@ def _run_single_test(test, test_iterator):
 
                 except context.session.get_skip_exception_types():
                     pass
-                except INTERRUPTION_EXCEPTIONS:
```

Comment:
Why is this still needed?
All it did is firing the `test_interrupt` hook, but now we're firing it once the interruption is caught. What am I missing?